lib/internal/child_process.js

```diff
@@ -655,6 +655,16 @@ function setupChannel(target, channel) {
     if (message === undefined)
       throw new ERR_MISSING_ARGS('message');
 
+    // Non-serializable messages should not reach the remote
+    // end point; as any failure in the stringification there
+    // will result in error message that is weakly consumable.
+    // So perform a sanity check on message prior to sending.
+    const ret = JSON.stringify(message);
+    if (ret == null) {
```

Comment:
`ret` can never be `null`. If `null` is passed through as message `JSON.stringify()` would return the string `'null'`.

JSON.stringify() either returns `undefined` or a `string` or it throws.